tests/foreman/api/test_syncplan.py

```diff
@@ -481,3 +600,11 @@ def test_delete_sync_plan_with_one_synced_product(self):
         @Feature: SyncPlan
 
         """
+        syncplan = entities.SyncPlan(organization=self.org).create()
+        product = entities.Product(organization=self.org).create()
+        entities.Repository(product=product).create()
+        syncplan.add_products([product.id])
+        product.sync()
```

Comment:
I might be wrong, as I'm not very familiar with this functionality, but as I understand synchronize command for product actually synchronizes all repositories in it. 
Repositories, created with nailgun by default are of 'yum' type with [following URL](https://github.com/SatelliteQE/nailgun/blob/master/nailgun/entities.py#L64):
```python
_FAKE_YUM_REPO = 'http://inecas.fedorapeople.org/fakerepos/zoo3/'
```
That's probably the answer to your question. Please, correct me if i'm wrong in smth.